pkgs/top-level/python-packages.nix

```diff
@@ -30195,4 +30233,162 @@ in modules // {
     };
   };
 
+  homeassistant = buildPythonApplication rec {
+    name = "${pname}-${version}";
+    pname = "homeassistant";
+    version = "0.30.2";
+
+    disabled = !isPy3k;
+
+    outputs = [ "out" ];
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/h/homeassistant/${name}.tar.gz";
+      sha256 = "1w4ngbx5wg0qjhanwcfp49gkmpmhvlqk3xy4pjv55mpzxc3l6xhf";
+    };
+
+    # I cannot get the tests to run
+    doCheck = false;
+
+    buildInputs = with self; [ wrapPython ];
+
+    propagatedBuildInputs = with self; [
+      cherrypy
+      jinja2
+      netdisco
+      pytz
+      pyyaml
+      requests2
+      sqlalchemy
+      static3
+      voluptuous
+      werkzeug
+    ];
+
+    preConfigure = ''
+      # typing is part of stdlib in python 3.5+
```

Comment:
does that mean the package should only be available for Python >= 3.5? `disabled = pythonOlder "3.5";`